cmake/README.md

```diff
@@ -0,0 +1,28 @@
+## The CMake for Mbed OS
+
+Until further notice, CMake for Mbed OS is still in the development.  
+
+### Supported targets
+
+Only a limited set of targets is supported at the moment.
+
+These targets should work:
+- K64F 
+- DISCO_L475VG_IOT01A 
+- NRF52840_DK 
+- WIO_3G
+- K66F
+
+### Supported toolchains
+
+ARMClang6 and Gcc Arm toolchains are supported.
+
+### Examples
+
+Supported examples can be identified by the presence of a top level `CMakelists.txt` input source file. 
+
+### Known issues
+
+- ARMClang6: the assembler does not include definitions. It will be fixed in the upcoming release (>3.18.2)
```

Comment:
Maybe a link to the issue you raised? It will show readers what to do to fix it on their system so they can build.

Reply:
fixing now